release-notes/SupportedFeatures-v1.0.0.md

```diff
@@ -19,7 +19,7 @@ Supported features
 |Transports    |Http                   |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
```

Comment:
Can we add a link pointing back to the release notes underneath the title (line 3) like what we had for [RC2](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v1.0.0-rc2.md#wcf-features-in-rc2)?